api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2383,6 +2386,221 @@ public void copyObject(String bucketName, String objectName, Map<String,String>
     }
   }
 
+  /**
+   * Create an object by concatenating a list of source objects using server-side copying.
+   * </p>
+   * <b>Example:</b><br>   *
+   * <pre>
+   * {@code minioClient.composeObject(String bucketName, String objectName,
+   * List<ComposeSource> composeSource, Map<String,String> userMetaData,
+   * ServiceConfigurationError sse );}
+   * </pre>
+   *
+   * @param bucketName
+   *          Destination Bucket to be created upon compose.
+   * @param objectName
+   *          Destination Object to be created upon compose.
+   * @param sources
+   *          List of Source Objects used to compose Object.
+   * @param headerMap
+   *          User Meta data.
+   * @param sse
+   *          Server Side Encryption.
+   *
+   * @throws InvalidBucketNameException  upon invalid bucket name is given
+   * @throws NoSuchAlgorithmException
+   *           upon requested algorithm was not found during signature calculation
+   * @throws InsufficientDataException  upon getting EOFException while reading given
+   *           InputStream even before reading given length
+   * @throws IOException                 upon connection error
+   * @throws InvalidKeyException
+   *           upon an invalid access key or secret key
+   * @throws NoResponseException         upon no response from server
+   * @throws XmlPullParserException      upon parsing response xml
+   * @throws ErrorResponseException      upon unsuccessful execution
+   * @throws InternalException           upon internal library error
+   * @throws InvalidArgumentException    upon invalid value is passed to a method.
+   * @throws InvalidResponseException    upon a non-xml response from server
+   */
+
+  public void composeObject(String bucketName, String objectName, List<ComposeSource> sources,
+      Map<String,String> headerMap, ServerSideEncryption sse)
+    throws InvalidBucketNameException, NoSuchAlgorithmException, InsufficientDataException, IOException,
+    InvalidKeyException, NoResponseException, XmlPullParserException, ErrorResponseException,
+    InternalException, InvalidArgumentException, InvalidResponseException {
```

Comment:
`InvalidResponseException` is declared here as one of the possible exceptions `composeObject` can throw, but the `API.md` file does not explain about it, i.e. it is missing in the list under `Listed Exceptions:`.

Reply:
done.